### ISSUE TITLE
Show coloured output on invoke tasks

### ISSUE THREAD

**Issue by jhonatan-lopes** [MozillaFoundation/foundation.mozilla.org#11768]
Invoke tasks are currently not outputting colour to the terminal, most likely because tasks think they are being run outside of a terminal. To fix this, a [`pty=True` should be added to the `run` command of `invoke` tasks](https://github.com/pyinvoke/invoke/issues/256#issuecomment-115339895):

```py
ctx.run("my cmd", pty=True)
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/TP1-240)

**Comment by AdalbertoMoz** [comment]
The ticket states "Invoke tasks are currently not outputting color to the terminal," but they actually are. The `pty=True` setting in the ctx command execution was added in PR #1556 (from 6 years ago) to enable color output:

https://github.com/MozillaFoundation/foundation.mozilla.org/blob/5f66e3f3a4f35e197a3909c8ef07cc2a06d78228/tasks.py#L11-L16

This `pty=True` setting works for Unix-like systems to ensure proper rendering of command outputs, including colors. This setting is crucial for commands requiring a pseudo-terminal (PTY) for interactive processes.

However, Windows does not support PTYs natively. To handle output on Windows, the `PYTHONUNBUFFERED `environment variable is set to `True`, avoiding potential issues with PTYs.

After some investigation, I found that pseudo-terminals in Windows can be complex but can be managed with the `pywinpty` package. However, I couldn't get this to work yet. Before investing more time in this solution, we should consider if it's necessary.

Given that almost everyone is using Unix-like systems and our deployment is on Unix-based Heroku dynos, is it worth investing more time to enable color output on Windows? @simont-cr @danielfmiranda
